src/com/owncloud/android/ui/activity/Preferences.java

```diff
@@ -308,6 +309,8 @@ public boolean onPreferenceClick(Preference preference) {
     @Override
     protected void onPause() {
         super.onPause();
+        
+        PinCheck.setUnlockTimestamp();
```

Comment:
This is not enough to protect the app when the user press HOME while showing the Settings activity.